packages/drag-drop/demo/~patterns/stories/SortableListsStory.tsx

```diff
@@ -0,0 +1,354 @@
+
+interface IColumnProps {
+  id: UniqueIdentifier;
+  items: ISortableItem[];
+}
+
+interface ISortableItem {
+  label: string;
+  caption: string;
+  id: UniqueIdentifier;
+}
+
+interface ISortableItemProps {
+  data: ISortableItem;
+  activeItem?: ISortableItem | null;
+  isOverlay?: boolean;
+  isGrabbed?: boolean;
+  tabIndex?: number;
+}
+
+type IColumns = Record<UniqueIdentifier, ISortableItem[]>;
+
+interface IArgs {
+  columns: IColumns;
+}
+
+const StyledSortablesContainer = styled.div`
+  display: flex;
+  gap: 16px;
+  max-width: 600px;
+`;
+
+export const getAnnouncements = (
+  getPosition: (id: UniqueIdentifier | undefined) => number,
+  count: number | undefined
+): Announcements => ({
+  onDragStart({ active }) {
+    return `Picked up sortable item ${active.id}. Sortable item ${
+      active.id
+    } is in position ${getPosition(active.id as UniqueIdentifier)} of ${count}`;
+  },
+  onDragOver({ active, over }) {
+    return `Sortable item ${active.id} was moved into position ${getPosition(
+      over?.id
+    )} of ${count}`;
+  },
+  onDragEnd({ active, over }) {
+    return `Sortable item ${active.id} was dropped at position ${getPosition(
+      over?.id
+    )} of ${count}`;
+  },
+  onDragCancel({ active }) {
+    return `Dragging was cancelled. Sortable item ${active.id} was dropped.`;
+  }
+});
+
+function findColumn(
+  id: UniqueIdentifier | undefined,
+  columns: IColumns
+): UniqueIdentifier | undefined {
+  if (!id) return undefined;
+
+  if (id in columns) {
+    return id;
+  }
+
+  return Object.keys(columns).find(cId => {
+    return columns[cId].findIndex(item => item?.id === id) > -1;
+  });
+}
+
+export const SortableListsStory: Story<IArgs> = ({ columns: defaultColumns }: IArgs) => {
+  const [columns, setColumns] = useState<IColumns>(defaultColumns);
+
+  // state fallback for cancelled drag
+  const [snapshot, setSnapshot] = useState<IColumns | null>(null);
+
+  // active drag item pointer - item & column
+  const [activeId, setActiveId] = useState<UniqueIdentifier | null>(null);
+  const [activeColumnId, setActiveColumnId] = useState<UniqueIdentifier | null>(null);
+
+  // Overlay ref to move focus when dragging
+  const overlayRef = useRef<HTMLDivElement>(null);
+
+  // derive the active item if activeId/activeColumnId is present
+  const activeItem =
+    activeId && activeColumnId
+      ? columns[findColumn(activeId, columns)!].find(item => item.id === activeId)
+      : null;
+
+  // DndKit interaction sensors
+  const sensors = useSensors(
+    useSensor(PointerSensor),
+    useSensor(TouchSensor),
+    useSensor(KeyboardSensor, {
+      coordinateGetter: sortableKeyboardCoordinates
+    })
+  );
+
+  const onDragStart = ({ active }: DragStartEvent) => {
+    const columnId = findColumn(active.id, columns) as UniqueIdentifier;
+
+    setActiveId(active.id);
+    setActiveColumnId(columnId);
+    setSnapshot(columns);
+  };
+
+  const onDragOver = useMemo(
```

Comment:
nit: (also not sure if this is the right place to insert comment) 

When dragging an item over the Dropzone, the `<Dropzone.Message>` should change to "Drop item here". This matches our patterns and could be a helpful implementation detail on changing the dropzone message during `onDragOver`

Reply:
I can try to add this. :)

Reply:
Added. Latest deploy in description.